CMakeLists.txt

```diff
@@ -18,9 +18,9 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake
 include(GitExternal)
 
 set(VERSION_MAJOR "0")
-set(VERSION_MINOR "4")
+set(VERSION_MINOR "5")
 set(VERSION_PATCH "0")
-set(VERSION_ABI 3) # OFF or ABI version
```

Comment:
The API changed and here also ABI changed ? Especially for the data sources that are based on this Livre "has to" be recompiled ?